src/icons/Coin.tsx

```diff
@@ -0,0 +1,33 @@
+import CommonIconsProps from 'icons/CommonIconsProps'
+import classnames, { stroke, strokeWidth } from 'classnames/tailwind'
+
+export default function ({ inheritStrokeColor }: CommonIconsProps) {
+  const commonPathStyles = classnames(
+    stroke(inheritStrokeColor ? 'stroke-inherit' : 'stroke-primary-background'),
```

Comment:
`inherit` is needed here, because we change icon colour when selected:
![image](https://user-images.githubusercontent.com/49658988/187036705-937d15bb-bb9f-44c8-b407-650f6c502645.png)